src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml

```diff
@@ -18,6 +18,7 @@ limitations under the License.
   <domain name="CHIP"/>
 
   <enum name="AccessControlEntryPrivilegeEnum" type="enum8">
+    <cluster code="0x0752"/> <!-- Joint Fabric Datastore -->
```

Comment:
In spec terms, you can't use a type from one cluster in another cluster: all type definitions in the spec are either global, or scoped to a specific cluster.

I know you can write the asciidoc for it, but there is no machinery in the spec for handling resulting name collisions...

So the spec is broken here and needs fixing.

Of course this SDK implementation is not actually matching the spec, in the sense that this is more of a "there are two enums that we will write the XML for once, but you will actually get two separate enums", which is not what the spec is trying (incorrectly) for....

Reply:
Copied the enum definition to the JF datastore cluster and submitted the update in the most recent change set.

Reply:
I filed https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11471 to track this on the spec side.